extensions.json

```
@@ -972,6 +972,12 @@
       "id": "openfl.lime-vscode-extension",
       "repository": "https://github.com/openfl/lime-vscode-extension"
     },
+    {
+      "id": "paiqo.databricks-vscode",
+      "repository" : "https://github.com/paiqo/Databricks-VSCode",
+      "checkout": "v0.6.0",
+      "version": "0.6.0"
```

I see that this repository even has an official `.vsix` release. In these cases, we prefer to re-publish that directly, instead of attempting to repackage the extension ourselves:

```suggestion
      "download" : "https://github.com/paiqo/Databricks-VSCode/releases/download/v0.6.0/databricks-vscode-0.6.0.vsix",
      "version": "0.6.0"
```